web/web.go

```diff
@@ -110,6 +111,40 @@ func (s *PrometheusStatus) ApplyConfig(conf *config.Config) bool {
 	return true
 }
 
+// PrometheusVersion contains various build information about Prometheus
+type PrometheusVersion struct {
+	Version   string
```

Comment:
That's what I've said in the description, but thanks I didn't though about specifying json names in the struct. I will fix this.